src/overlays/actors/ovl_En_Tg/z_en_tg.c

```diff
@@ -76,37 +91,200 @@ static DamageTable D_809901F8 = {
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
 };
 
-#endif
+static AnimationInfoS sAnimations = { &gHoneyAndDarlingIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 };
 
-extern ColliderCylinderInit D_809901C0;
-extern CollisionCheckInfoInit2 D_809901EC;
-extern DamageTable D_809901F8;
+/**
+ * This function is always called with unusedExtraOffset = 0.
+ * The only animation used is the Idle (stationary) animation.
+ */
+void EnTg_ChangeAnimation(SkelAnime* skelAnime, AnimationInfoS* animation, s16 unusedExtraOffset) {
```

Comment:
```suggestion
void EnTg_ChangeAnim(SkelAnime* skelAnime, AnimationInfoS* animationInfo, s16 animIndex) {
```
We have started to standardize these functions.